File: python/high-scores/high_scores.py
class HighScores(object):
    def __init__(self, scores):
        self.scores = scores

    def latest(self):
        return self.scores[-1]

    def personal_best(self):
        return self.__rank()[0]

    def personal_top_three(self):
        return self.__rank()[:3]

    def __rank(self):
        return sorted(self.scores, reverse=True)

File: javascript/high-scores/high-scores.js
export class HighScores {
  constructor(scores) {
    this._scores = scores;
  }

  get scores() {
    return this._scores;
  }

  get latest() {
    return this.scores[this.scores.length - 1];
  }

  get personalBest() {
    throw new Error("Remove this statement and implement this function");
  }

  get personalTopThree() {
    throw new Error("Remove this statement and implement this function");
  }
}
